### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by cwaku**
With regards to the project requirements:

- Initialised React app.
- Added React-Redux (`npm install react-redux`).
- Structured your application files.
- Added React Router and set two `<Route>`s and `<Link>`s for the app's navigation.

### CODE REVIEW

File: src/pages/Books.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+export default function Books() {
+  return (
+    <main>
+      <div className="books-container">
+        <p>No books added</p>
+        <button type="button">Remove</button>
+      </div>
+    </main>
+  );
```
Comment (TedLivist) [inline src/pages/Books.js]: - [ ] Kindly create a new component for adding a book. The component should contain inputs fields for `title` and `author` and a placeholder Add button (no functionality yet). Render the form component here.
- [ ] You are missing a component for rendering a single book. The component should take in book data (`title` and `author`) as a prop and render them. It should also have a placeholder Remove button. Then render that component here also.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @cwaku,

Good job so far!
✔️ Good documentation
✔️ React routes is set up
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Books.js (hunk 1 above)
Comment (cwaku) [reply src/pages/Books.js]: @TedLivist  I honestly thought that would be in the next project

Comment (TedLivist) [reply src/pages/Books.js]: Hi @cwaku not really. As indicated, you are initializing the components at this point in the project.

**Review by Emmyn5600 — APPROVED** [review]
Hi @cwaku 

Woo ow 👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**